verl/workers/rollout/vllm_rollout/vllm_rollout_spmd.py

```diff
@@ -579,9 +606,20 @@ async def update_weights(self, weights: Generator[tuple[str, torch.Tensor], None
         """
         from verl.utils.vllm.patch import patch_vllm_moe_model_weight_loader
 
-        model = self.inference_engine.worker.model_runner.model
+        model_runner = self.inference_engine.worker.model_runner
+        model = model_runner.model
         patch_vllm_moe_model_weight_loader(model)
-        model.load_weights(weights)
+
+        # Add the FP8 related logic here as sharding manager has been deprecated.
```

Comment:
We're going to switch rollout from SPMD mode to server mode, please also modify `vLLMAsyncRollout`, which is used in server mode.